src/lib/nvme_pt_ioctl.c

```diff
@@ -200,42 +198,57 @@ static int get_opal_auth_uid(enum SED_AUTHORITY auth)
 	return sed2opal_map[auth];
 }
 
-static void check_tper_feat(void *feat)
+static void check_tper_feat(struct sed_opal_level0_discovery *discv,
```

Comment:
function naming is a little bit confusing: it doesn't check anything it just perform memcpy

Reply:
yup, will put it in another commit